marge/merge_request.py

```diff
@@ -39,19 +39,18 @@ def fetch_by_iid(cls, project_id, merge_request_iid, api):
     def fetch_assigned_at(cls, user, api, merge_request):
         assigned_at = 0
         all_discussions = api.collect_all_pages(
-            GET('/projects/{project_id}/merge_requests/{merge_requests_id}/discussions'.format(
-                project_id=merge_request.get('project_id'),
-                merge_requests_id=merge_request.get('iid')
-            )))
-        match_body = 'assigned to @{username}'.format(username=user.username)
+            # pylint: disable=line-too-long
+            GET(f"/projects/{merge_request.get('project_id')}/merge_requests/{merge_request.get('iid')}/discussions")
```

Comment:
Almost worth extracting these `merge_request.get('project_id')` calls to local variables in this method or WDYT?

Reply:
```diff
diff --git a/marge/merge_request.py b/marge/merge_request.py
index 6a7bde9..1b0b9e2 100644
--- a/marge/merge_request.py
+++ b/marge/merge_request.py
@@ -38,9 +38,11 @@ class MergeRequest(gitlab.Resource):
     @classmethod
     def fetch_assigned_at(cls, user, api, merge_request):
         assigned_at = 0
+        project_id = merge_request.get('project_id')
+        merge_request_iid = merge_request.get('iid')
+
         all_discussions = api.collect_all_pages(
-            # pylint: disable=line-too-long
-            GET(f"/projects/{merge_request.get('project_id')}/merge_requests/{merge_request.get('iid')}/discussions")
+            GET(f'/projects/{project_id}/merge_requests/{merge_request_iid}/discussions')
         )
         match_body = f'assigned to @{user.username}'
         for discussion in all_discussions:
```